containers/transcode/transcode.sh

```diff
@@ -30,4 +33,4 @@ export AWS_ACCESS_KEY_ID=$(echo $role | jq -r .Credentials.AccessKeyId)
 export AWS_SECRET_ACCESS_KEY=$(echo $role | jq -r .Credentials.SecretAccessKey)
 export AWS_SESSION_TOKEN=$(echo $role | jq -r .Credentials.SessionToken)
 
-aws s3 cp output s3://"$STATE_MACHINE_DESTINATION_BUCKET_NAME"/"$STATE_MACHINE_DESTINATION_OBJECT_KEY"
+aws s3 cp output s3://"$STATE_MACHINE_DESTINATION_BUCKET_NAME"/"$STATE_MACHINE_DESTINATION_OBJECT_KEY" --content-type "$STATE_MACHINE_DESTINATION_PARAMETER_CONTENT_TYPE"
```

Comment:
So this is only content-type, not other metadata right?  May be worth a ticket for adding content-disposition (and other headers) support for transcode.